### PR TITLE
Added news in RSS feed and link to faq page

### PR DESCRIPTION
**Pull request by Nikhil-Ladha**
Signed-off-by: Nikhil-Ladha <nikhilladha1999@gmail.com>

**Description**
This PR adds `news` in the RSS feed, and also adds a link to the `faq` page in the footer. 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 15efbe83654cd7cfb186b1be32707922b70bf788 at: https://604fa05a812fe42fc16ecaff--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit a7650654f72a85c2b53c74b289216ff1eae9d725 at: https://604fa9c663dd94066090b324--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 5dc833fd1020618b37fe27c9bfd41f0b3bb7f4dc at: https://604fad73a376b309870088eb--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 90c8a9911c40ba3a87a94c1a0bac0288e66e52c2 at: https://604fb30e1dbc190fc1c75d3a--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit be17ce8c3b5135861add51c953f6af77698b8d84 at: https://604fb7c323b9bc12b091a9b9--layer5ng.netlify.app

File: gatsby-config.js [hunk 1]
```diff
@@ -62,8 +62,8 @@ module.exports = {
               {
                 allPosts: allMdx(
                   sort: { fields: [frontmatter___date], order: DESC }
-                  filter: { fields: { collection: { eq: "blog" } }, frontmatter: { published: { eq: true } } }
-                  limit: 15
+                  filter: { fields: { collection: { in: ["blog", "news"] } }, frontmatter: { published: { eq: true } } }
```
Comment (leecalcote) [inline gatsby-config.js:65]: Radical

File: gatsby-config.js [hunk 2]
```diff
@@ -85,7 +85,7 @@ module.exports = {
               }
             `,
             output: "/rss.xml",
-            title: "Layer5 Blogs"
+            title: "Layer5"
```
Comment (leecalcote) [inline gatsby-config.js:88]: Most excellent.

File: src/sections/General/Footer/index.js [hunk 3]
```diff
@@ -146,6 +146,11 @@ const Footer = () => {
                       Contact
                     </Link>
                   </li>
+                  <li>
+                    <Link className="category-link" to="/company/faq">
```
Comment (leecalcote) [inline src/sections/General/Footer/index.js:150]: Totally tubular.

**Review by leecalcote — APPROVED** [review]

**Pull request merged**